kahuna/public/js/search/results.js

```diff
@@ -95,6 +95,10 @@ results.controller('SearchResultsCtrl', [
 
         const ctrl = this;
 
+        ctrl.$onInit = () => {
+          ctrl.showSendToPhotoSales = () => $window._clientConfig.showSendToPhotoSales;
```

Comment:
showSendToPhotoSales could perhaps be a var/const rather than a function (but if makes not functional difference and probably minimal performance impact)